src/git.js

```diff
@@ -0,0 +1,42 @@
+import {exec} from "child_process";
+import {Promise} from "es6-promise";
+import logger from "./logger";
+
+
+const git = {
+  dry: false,
+
+  checkout(filePath) {
+    return new Promise((resolve, reject) => {
+      git.exec(`git checkout ${filePath}`, (err, stdout) => {
+        if (err) {
+          logger.error(`checkout '${filePath}'`, err);
+
+          return reject(err);
+        }
+
+        logger.success(`checkout '${filePath}'`);
+
+        return resolve(stdout);
+      });
+    });
+  },
+
+  disableDry() {
+    git.dry = false;
+  },
+
+  exec(cmd, cb) {
+    if (git.dry) {
+      exec("git status", cb);
```

Comment:
I like the idea of calling `git status` here in place of any destructive git commands to make sure git is callable.
Thoughts?